test/contracts/HCElection.aes

```diff
@@ -150,7 +150,7 @@ main contract HCElection =
       require(penalty > 0, "Penalty is expressed as a positive integer")
       require(reporter_percentage >= 0 && reporter_percentage =< 100, "reporter reward percentage must be between 0 and 100")
       put(state{rewards[height = []] @ rs = (to, (-penalty)) :: rs})
-      let reporter_reward = penalty * (100/reporter_percentage)
+      let reporter_reward = (penalty * reporter_percentage) / 100
```

Comment:
if anything, then (100*p) / r, right? Or did math change?
I mean, the change in code changes the computation, it's not just a rewrite?